README.md

```diff
@@ -184,6 +184,29 @@ You can configure per-request tags by defining `salestation.statsd.tags` in sina
   end
 ```
 
+## RSpec helpers
+
+
+To use Salestation RSpec matchers you first need to include them:
+
+```
+require 'salestation/rspec'
+
+RSpec.configure do |config|
+  config.include Salestation::RSpec::Matchers
+end
+```
+
+After that it's possible to match against error results like this:
+
+```
+expect(result).to be_failure
+  .with_conflict
+  .containing(hash_including(message: 'Oh noes'))
```

Comment:
maybe just `with` or `contain_payload`
looking at some of the rspec matchers, maybe something like this would be more RSpec like:
```
expect(result).to be_failure
  .and.have_status(:conflict)
  .and.contain_payload(message: 'Oh noes')
```
or
```
expect(result).to be_failure
  .and.have_status(:conflict)
  .and.include_payload(hash_including(message: 'Oh noes'))
```
https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/built-in-matchers/have-attributes-matcher
https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/composing-matchers#composing-matchers-with-%60contain-exactly%60
https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/built-in-matchers/include-matcher

Reply:
`.with_conflict` is actually alias for `.with(Salestation::App::Errors::Conflict)`.

Reply:
but suggestions are welcome for the design. I just cannot use the RSpec original `.and` composer. I could define it myself but that might be even more confusing because then it's a completely different `and`.

Reply:
Will `expect(result).to be_failure.with_conflict` work on its own or it always requires `containing`?

Reply:
It will work independently as well. `expect(result).to be_failure` is valid, `expect(result).to be_failure.with_conflict` is also valid, and also with `.containing` or `.based_on` can be added to the end as well.